frontend/public/components/configmap-and-secret-data.jsx

```diff
@@ -1,14 +1,36 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { Heading } from './utils';
+import { Heading, CopyToClipboard } from './utils';
 
-export const ConfigMapData = ({data}) => {
+const MaskedData = () => {
+  return <React.Fragment>
+    <span className="sr-only">Value hidden</span>
+    <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
+  </React.Fragment>;
+};
+
+const Icon = ({showSecret}) => {
```

Comment:
I'm not sure it's really worth pulling just the icon out here because you have to check the `showSecret` value twice... I'd keep the icon with the text.